tests/embed/test_mpca.py

```diff
@@ -59,19 +59,16 @@ def test_mpca(var_ratio, n_components):
 def test_mpca_against_baseline():
     x = gait["fea3D"].transpose((3, 0, 1, 2))
     baseline_proj_mats = [baseline_model["tUs"][i][0] for i in range(baseline_model["tUs"].size)]
+    baseline_mean = baseline_model['TXmean']
     mpca = MPCA(var_ratio=0.97)
     x_proj = mpca.fit(x).transform(x)
-    baseline_proj_x = multi_mode_dot(x, baseline_proj_mats, modes=[1, 2, 3])
-    # check whether the output shape is consistent with the baseline output by keeping the same variance ratio 97%
-    testing.assert_equal(x_proj.shape, baseline_proj_x.shape)
+    testing.assert_allclose(baseline_mean, mpca.mean_)
+    baseline_proj_x = multi_mode_dot(x - baseline_mean, baseline_proj_mats, modes=[1, 2, 3])
+    # check whether the output embeddings is close to the baseline output by keeping the same variance ratio 97%
+    testing.assert_allclose(x_proj ** 2, baseline_proj_x ** 2, rtol=0.00001)
```

Comment:
The tolerance value is used more than once so it may be better to set it as a "global" variable at the top (and in future, maybe in `conftest.py`)

Reply:
will change